pkg/controller/servicebindingrequest/olm.go

```diff
@@ -198,11 +198,11 @@ func buildCRDDescriptionFromCRD(crd *unstructured.Unstructured) (*olmv1alpha1.CR
 	}
 
 	if specDescriptor != nil {
-		crdDescription.SpecDescriptors = append(crdDescription.SpecDescriptors, *specDescriptor)
+		crdDescription.SpecDescriptors = append(crdDescription.SpecDescriptors, specDescriptor...)
```

Comment:
how about to rename the arg name to plural form, i.e. `specDescriptors`?

Reply:
Good point, done https://github.com/redhat-developer/service-binding-operator/pull/371/commits/e75799f9324edc0fd8a02788c6397186c37c0a1f